packages/cli/lib/services/local-integration.service.ts

```diff
@@ -60,7 +60,8 @@ class IntegrationService implements IntegrationServiceInterface {
                                 throw new Error(`Module '${moduleName}' is not allowed`);
                         }
                     },
-                    Buffer
+                    Buffer,
+                    setTimeout
```

Comment:
at some point I would like to consolidate the dry-run and non dry-run code, since there are now both using node.vm